src/main/java/core/basesyntax/model/Truck.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+import core.basesyntax.Machine;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Common Truck, You can do it");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Ohh Truck, try again");
```

Comment:
Similarly, the message in 'stopWork' should be informative and indicate that it is a Truck which is stopping its work. Consider changing it to something like 'Truck is stopping its work'.